airflow/providers/databricks/operators/databricks.py

```diff
@@ -590,7 +590,7 @@ def __init__(
         python_params: Optional[List[str]] = None,
         jar_params: Optional[List[str]] = None,
         spark_submit_params: Optional[List[str]] = None,
-        python_named_parameters: Optional[Dict[str, str]] = None,
+        python_named_params: Optional[Dict[str, str]] = None,
```

Comment:
Why change this? The old argument name sounds perfectly fine to me and it’s not worthwhile to break backward compatibility IMO.

Reply:
This is to match Databricks Jobs API changes (really was a problem with published spec). This parameter most probably wasn't used yet, otherwise we would get an error report because API. I can keep it compatible with previous versions by adding a new parameters, and marking old one as deprecated...

Reply:
If it was not matching the expected value, this is a bugfix, not a breaking change.